input/blog/2020-12-20-cake-v1.0.0-rc0002-released.md

```diff
@@ -0,0 +1,117 @@
+﻿---
+title: Cake v1.0.0-rc0002 released
+category: Release Notes
+author: devlead
+---
+
+Version 1.0.0-rc0002 of Cake has been released.
+
+This is the second, and probably last, pre-release version of Cake 1.0. Focus in this release has been to bring [Cake Frosting](/docs/running-builds/runners/cake-frosting) up to speed, align it with the Cake script runners and bug fixes.
+
+### Major new features
+
+* Frosting NuGet tool resolution
+* Align Cake Frosting CLI with Cake script runners
+* Bug fixes
+* Addin recommended Cake version set to 1.0
+
+### Why Release Candidate?
+
+We've chosen to release 1.0 as a pre-release to NuGet and Chocolatey, so we and the community can more easily test and ensure that our 1.0 release will be as stable as possible.
+It will also enable us to see if there are any steps you need to take to adapt your scripts for 1.0, and for us to see if there're any unintended breaking changes, by testing as many addins, modules, and recipes as possible.
+
+### Breaking changes in Cake Frosting
+
+`CakeHostBuilder` has been removed.
+With Cake.Frosting 1.0 `CakeHost` can be used directly to create the `CakeHost` object.
+
+With Cake.Frosting 0.38.x:
+
+```csharp
+// Create the host.
+var host =
+    new CakeHostBuilder()
+       .WithArguments(args)
+       .UseStartup<Program>()
+       .Build();
+
+// Run the host.
+return host.Run();
+```
+
+With Cake.Frosting 1.0:
+
+```csharp
+// Create and run the host.
+return
+    new CakeHost()
+        .UseContext<BuildContext>()
+        .Run(args);
+```
+
+### Addin recommended Cake version set to 1.0
+
+In preparation for the 1.0 release, we've now set the recommended version of [Cake.Core](https://www.nuget.org/packages/Cake.Core/1.0.0-rc0002) for addins to target to 1.0.
+
+This won't break the build but you might see warnings like below when Addins/Modules are loaded.
+
+```bash
+The assembly 'Cake.DotNetTool.Module, Version=0.4.0.0, Culture=neutral, PublicKeyToken=null'
+is referencing an older version of Cake.Core (0.33.0).
+```
+
+Addin authors can start preparing for the 1.0 release by starting to build against [Cake.Core](https://www.nuget.org/packages/Cake.Core/1.0.0-rc0002) version 1.0.0-rc0002.
```

Comment:
AFAIK looking at the code, this won't help regarding the warning. Addin authors can't do anything at the moment to get rid of the warning. I think the message which we should make clear is that this warning stays until 1.0 is out and addin authors have updated their addins to built against 1.0.

Reply:
Doesn't rc2 have 1.0.0 as file version?
Haven't looked at code just now, but curious why targeting rc2 won't work.

Reply:
Yes, you're correct